src/main/java/core/basesyntax/dataservices/DataReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.dataservices;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataReaderImpl implements DataReader {
+    @Override
+    public List<String> read(String fileName) {
+        List<String> transactions = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
+            reader.readLine();
+            String line;
+            while ((line = reader.readLine()) != null) {
+                transactions.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
When catching an IOException, it's best to provide a more informative error message, such as including the file path that caused the exception.